docroot/modules/humsci/hs_bugherd/config/install/rest.resource.hs_bugherd_resource_jira.yml

```diff
@@ -1,4 +1,4 @@
-uuid: eddf0d9a-b688-4bf5-a170-ce0b622856c3
+uuid: e0b5cb53-6277-4923-b49e-8094974352e9
```

Comment:
Do you want a UUID in the module?

Reply:
:man_shrugging: i'm gone back and forth on it. since its the product i wasnt sure if it was a big concern.

Reply:
Future Mike will be thankful for having these removed. I'm 95% sure removing the UUID is the standard practice.

Reply:
Nice, past Mike is not always a jerk eh?

Reply:
he's got his moments